torch/testing/_internal/schema_check_tensor.py

```diff
@@ -0,0 +1,52 @@
+import torch
+from torch.utils._pytree import tree_map
+
+ops = []
+
+class SchemaCheckTensor(torch.Tensor):
```

Comment:
Could you add some brief documentation about what this class does and how it is to be used?